.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+## Description
+
+<!-- Before creating this PR, make sure to thoroughly follow the contributing guide. -->
+<!-- Add a description of the PR changes in this section. -->
+
+REPLACE ME
+
+
+## Tasks
+
+<!-- Once the PR has been created, check boxes as appropriate. -->
+
+* [ ] The [contributing guide](/.github/CONTRIBUTING.md) has been read and followed.
+* [ ] The samples added / modified have been fully tested.
+* [ ] Workflow files have been added / modified, if applicable.
+* [ ] Region tags have been properly added, if new samples.
```

Comment:
*Suggestion:*
I assume some contributors won't know what a "region tag" is.
```suggestion
* [ ] [Region tags](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/discussions/878) have been properly added, if new samples.
```
Feel free to ignore this suggestion or use a different link.

Reply:
Thought about it but I think it's best to keep all source of truth / docs / details in the "contributing.md" file (which the PR template points to). My answer to "what if someone doesn't know what a region tag is" would be "then they haven't read the contributing guide, which is the first checkbox they're supposed to check"